app/components/profile-page/testimonial-pic.js

```diff
@@ -1,5 +1,5 @@
 import Component from '@ember/component';
 
 export default Component.extend({
-  classNames: [ 'testimonial-pic' ]
+  classNames: [ 'testimonial-pic-kim-and-dan' , 'testimonial-pic-lex-and-marjon' , 'testimonial-pic-najja' ]
```

Comment:
Same as image-page, I created three different classNames for the testimonial pics. Please let me know if there is a better ways to deal with this component usage. Thanks